src/main.d

```diff
@@ -690,7 +690,7 @@ int main(string[] args)
 			auto logMonitorCounter = 0;
 			auto fullScanCounter = 0;
 			bool fullScanRequired = true;
-			bool syncListConfiguredOverride = false;
+			bool syncListConfiguredOverride = true;
```

Comment:
This change looks strange, why do you default to true now?

Reply:
This is so that, if sync_list is enabled, a full scan is done at 1st check, rather than waiting until 10th check. This then captures correctly any 'delete' action as well that is only sent once.